src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -409,20 +409,20 @@ namespace AppInstaller::CLI::Workflow
         context <<
             Workflow::SelectInstaller <<
             Workflow::EnsureApplicableInstaller <<
-            Workflow::ManageDependencies <<
+            Workflow::GetInstallerDependencies << 
+            Workflow::ReportDependencies <<
             Workflow::InstallPackageInstaller;
     }
 
-    void ManageDependencies(Execution::Context& context)
+    void GetInstallerDependencies(Execution::Context& context)
     {
         if (Settings::ExperimentalFeature::IsEnabled(Settings::ExperimentalFeature::Feature::Dependencies))
         {
             const auto& installer = context.Get<Execution::Data::Installer>();
             if (installer && installer->Dependencies.HasAny())
             {
                 context.Add<Execution::Data::Dependencies>(installer->Dependencies);
-                context.Reporter.Info() << "This package requires the following dependencies:" << std::endl;
-                context << Workflow::ReportDependencies;
+                context.Reporter.Info() << Resource::String::InstallAndUpgradeCommandsReportDependencies << std::endl;

```

Comment:
It should not be reporting anything either; the "determine" step from the other comment should be simply a transform that collects the dependencies and places them on the context.

If you have different header statements for different flows, consider using a workflow task object that is constructed with the desired string resource for the output.